egs2/voxceleb/spk1/conf/train_RawNet3.yaml

```diff
@@ -0,0 +1,30 @@
+# RawNet3 reproduce recipe configuration.
+encoder: rawnet3
+#encoder_conf:  # uncomment for customized arguments
+
+#model_conf:    # uncomment for customized arguments
+
+frontend: raw
+
+projector: rawnet3
+projector_conf:
+  input_size: 3072
+  output_size: 256
+
+
+preprocessor: spk
+preprocessor_conf:
+  utt2spk: dump/raw/voxceleb2_dev/utt2spk
+  spk2utt: dump/raw/voxceleb2_dev/spk2utt
+  target_duration: 3.0
+  sr: 16000
+  num_eval: 1
+
+loss: aamsoftmax
+loss_conf:
+  nOut: 256
+  nClasses: 5994
```

Comment:
How do you decide num of classes here? Can we possible automatically infer this in the template?

FYI, we have a utt2sid stages (at stage 2+) in TTS where we can convert speakers into ids, then we can easily get the number of speaker during training.

Reply:
I'm closing this since I connected and replied in your previous comment on utt2spk